app/models/local_authority.rb

```diff
@@ -32,6 +32,8 @@ class LocalAuthority < ApplicationRecord
   validates :gias_code, uniqueness: true, allow_nil: true
   validates :gss_code, uniqueness: true, allow_nil: true
 
+  has_many :local_authority_postcodes, foreign_key: :gss_code, primary_key: :gss_code
```

Comment:
```suggestion
  has_many :postcodes, foreign_key: :gss_code, primary_key: :gss_code, class_name: "LocalAuthorityPostcode"
```

Just to make using this association slightly nicer for developers 😄